CODE_OF_CONDUCT.md

```diff
@@ -58,7 +58,7 @@ Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported by contacting the project team at pawamoy@pm.me. All
 complaints will be reviewed and investigated and will result in a response that
 is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
+obligated to maintain confidentiality concerning the reporter of an incident.
```

Comment:
I'll update my project template to the latest version of the Covenant code of conduct. Don't bother reverting those though, it's fine.